app/controllers/campaigns_controller.rb

```diff
@@ -176,6 +177,15 @@ def articles_csv
     end
   end
 
+  def all_edits
```

Comment:
This should have a test.

Reply:
Great feedback. I will also work on this.